charts/datahub/subcharts/datahub-gms/templates/hazelcastService.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ printf "%s-%s-%s" .Release.Name (regexReplaceAll "\\W+" .Values.global.datahub.version "-") "hazelcast-svc" | trunc 63 | trimSuffix "-" }}
+  name: {{ printf "%s-%s-%s" .Release.Name (regexReplaceAll "[^-a-z0-9]+" .Values.global.datahub.version "-") "hazelcast-svc" | trunc 63 | trimSuffix "-" }}
```

Comment:
Additionally, the value here must be aligned in the deployment:

```
            {{- if gt .Values.replicaCount 1.0}}
            - name: SEARCH_SERVICE_CACHE_IMPLEMENTATION
              value: "hazelcast"
            - name: SEARCH_SERVICE_HAZELCAST_SERVICE_NAME
              value: {{ printf "%s-%s-%s" .Release.Name (regexReplaceAll "[^-a-z0-9]+" .Values.global.datahub.version "-") "hazelcast-svc" | trunc 63 | trimSuffix "-" }}
            {{- end}}
```

in https://github.com/sgomezvillamor/datahub-helm/blob/158ae1e99dfce4d0fff248c6786816cb7b99ce75/charts/datahub/subcharts/datahub-gms/templates/deployment.yaml#L126